grafana/README.md

```diff
@@ -0,0 +1,81 @@
+Grafana Dashboard
+=================
+
+This folder contains docker compose configuration to configure
+grafana/influxdb/telegraf to query (every 10 seconds) your AQI
+monitors, and to chart the AQI over time.
+
+Setup
+-----
+
+1. Edit `docker-compose.yaml` and change the IP address (192.168.0.2) to
```

Comment:
Could we use an `.env` file (or one specified with `env_file`) instead of editing the docker compose file?

Reply:
I was thinking... I'm not even sure why that IP address is in the docker-compose - it shouldn't be needed - I'll play about with that (I took the original docker-compose from someone else and have altered and fixed it in other ways)